src/staticobject.cpp

```diff
@@ -75,6 +75,7 @@ void StaticObjectList::serialize(std::ostream &os)
 }
 void StaticObjectList::deSerialize(std::istream &is)
 {
+	m_stored.clear();
```

Comment:
You'd have to clear m_active too...

... however, of you clear m_active, I suspect there are ramifications, as the active objects are known to at least one client...